src/Analyzer.Reports/SarifReportWriter.cs

```diff
@@ -205,11 +206,13 @@ private void ExtractRule(IEvaluation evaluation)
             return (isFileInRootPath, filePath);
         }
 
-        private FailureLevel GetLevelFromEvaluation(IEvaluation evaluation)
-        {
-            // The rule severity definition work item: https://github.com/Azure/template-analyzer/issues/177
-            return evaluation.Passed ? FailureLevel.Note : FailureLevel.Error;
-        }
+        private static FailureLevel GetLevelFromSeverity(Severity severity) =>
```

Comment:
Any concern about having a version of GetLevelFromSeverity defined in three places?

Reply:
I could add a utility class to the tests to reduce duplication there, I was just in a hurry.  That implementation should stay separate from the main code though, so that if a bug is introduced in SARIF processing, the tests don't simply use the same bug.

Reply:
Added shared utility function.